CONVENTIONS.md

```diff
@@ -0,0 +1,81 @@
+MCP Naming Conventions
+======================
+
+In order to guarantee a consistent style and theme across MCP names, the following conventions
+should be adhered to when mapping new things or proposing replacements for existing mappings.
+Please refer to this document whenever you do not know how to format or structure a name.
+
+Getting Started
+---------------
+ - **If you do not know what to name something then *do not name it*. Let someone else do so.**
+ - Names should reflect intent, *not* solely implementation. Check usages rather than looking only at the definition.
+   Also consider overrides and their context.
+ - Names should be concise and memorable. If information can be inferred from context (type, parameters etc.),
+   do not include it in the name. This especially applies to fields and methods in a class, where you do not
+   need to have the defining class's name in the member's name.
+ - Prefer clarification in the documentation comment over verbose names. Conversely, do not give fields,
+   methods, or method parameters needless comments that only repeat signature and name. Instead, use the documentation
+   for non-trivial examples or technical details.
+
+General
+-------
+ - All names use American English spelling.
+ - Acronyms should generally be written all-lowercase, while normal camel case rules still apply.
+   For instance, "identifier" becomes `id` as standalone word or at the beginning of a camel case phrase and
+   `Id` when used within a phrase.<br>
+   There are various exceptions to this rule which apply only when the word does not start with them:
```

Comment:
What about UUID?  Currently it's rather inconsistent with some uses being `UUID`, some being `Uuid`, and some not compacting it at all.  Mojang seems to use `UUID` for `Realms.getUUID` (but there's also `Realms.uuidToName`).

Reply:
`UniqueId` is what we tend to use.

Reply:
I've added an item about `UUID` -> `UniqueId`